### PR TITLE
Add new template: Deep Sea

### PR DESCRIPTION
**Pull request by rgvillanueva28**

### CODE REVIEW

File: deepsea/index.html [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        <section id="about">
+            <h2 class="text-gradient">About</h2>
+            <div>
+                <div>
+                    <img src="assets/logo.png" width="128" height="128" alt="Person Image" />
+                </div>
+                <ul>
+                    <li>
+                        <img class="about-icon" src="assets/user.svg" alt="person icon" />
+                        <p>Rane Gillian Villanueva</p>
```
Comment (jofftiquez) [inline deepsea/index.html:73]: We need to make the templates to have as less promotion as possible so if we could replace personal pics and names with placeholders, that would be better.

Comment (jofftiquez) [reply deepsea/index.html:73]: In short, no self promotion, I hope you understand. Thanks.

**Review by jofftiquez — CHANGES_REQUESTED** [review]

File: deepsea/index.html (hunk 1 above)
Comment (rgvillanueva28) [reply deepsea/index.html:73]: Yeah. No problem.

**Review by jofftiquez — CHANGES_REQUESTED** [review]
Just fix the conflicts and we're good to go <3 Thanks.

**Comment by rgvillanueva28** [comment]
Fixed it. Thank you.

**Pull request merged**